cmd/iso8583/describe.go

```diff
@@ -41,6 +41,33 @@ func Describe(paths []string, specName string) error {
 	return nil
 }
 
+func DescribeWithSpecFile(paths []string, specFileName string) error {
```

Comment:
I would extract core functionality into `func describe(paths, spec) error` and then inside `DescribeWithSpecFile` and `Describe` call this `describe(path, spec)` function.

Reply:
fixed